scripts/generate_dump

```diff
@@ -335,13 +512,19 @@ save_proc() {
 ###############################################################################
 # Dumps all fields and values from given Redis DB.
 # Arguments:
+#  DB id: id of DB for sonic-db-cli
```

Comment:
>DB id [](start = 3, length = 5)

The option has nothing to do with DB id. Why add?

Reply:
I changed the comments appropriately